interface/super/edit_list.php

```diff
@@ -1002,8 +1002,14 @@ function mysubmit() {
     }
   }
   else {
-    $res = sqlStatement("SELECT * FROM list_options WHERE " .
-      "list_id = '$list_id' ORDER BY seq,title");
+    /*
+     *  Add edit options to show or hide in list management
+     */
+    $res = sqlStatement("SELECT lo.* 
+                         FROM list_options as lo 
+                         right join list_options as lo2 on lo2.option_id = lo.list_id AND lo2.edit_options = 1
+                         WHERE lo.list_id = '{$list_id}' AND lo.edit_options = 1
+                         ORDER BY seq,title");
```

Comment:
Here I get the parent for the children by the list_id: 
` right join list_options as lo2 on lo2.option_id = lo.list_id AND lo2.edit_options = 1
`
If the option_id equals to the list_id it means lo2 is a parent. 


Here we're setting lo as the children:
`WHERE lo.list_id = '{$list_id}' AND lo.edit_options = 1`  

This creates a query where if the parent is off then the children wont show too.

If we use the query you suggested above, the children will show even if the parent is off.